src/module.ts

```diff
@@ -15,7 +15,7 @@ export default defineNuxtModule<PWAOptions>({
       source: null,
       sizes: [],
       fileName: 'icon.png',
-      targetDir: 'icons'
+      targetDir: 'icons',
```

Comment:
Dunno what you're using as VSCode extensions (if you're using VSCode) but ESLint should have automatically fixed this typo issue on file save (The ESLint configuration of the repository prevents these commas)